File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/points_exchange_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PointsExchangeVO:
    """积分兑换结果"""
    created_at: str = None
    updated_at: str = None
    record_id: str = None
    success: bool = None
    goods_name: str = None
    points_cost: int = None
    remaining_points: int = None
    quantity: int = None
    exchange_time: str = None
    status: str = None
    reward_content: str = None
    message: str = None

File: sdkwork-app-sdk-typescript/src/types/points-exchange-vo.ts
/** 积分兑换结果 */
export interface PointsExchangeVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 兑换记录ID */
  recordId?: string;
  /** 是否兑换成功 */
  success?: boolean;
  /** 商品名称 */
  goodsName?: string;
  /** 消耗积分 */
  pointsCost?: number;
  /** 剩余积分 */
  remainingPoints?: number;
  /** 兑换数量 */
  quantity?: number;
  /** 兑换时间 */
  exchangeTime?: string;
  /** 兑换状态 */
  status?: string;
  /** 到账内容 */
  rewardContent?: string;
  /** 提示消息 */
  message?: string;
}
